common/app/services/repositories.scala

```diff
@@ -127,6 +127,7 @@ trait Index extends ConciergeRepository with Collections {
   ): Future[Either[IndexPage, PlayResult]] = {
 
     val fields = if (isRss) rssFields else QueryDefaults.trailFieldsWithMain
+    val blocks = if (isRss) "body:latest:10" else ""
```

Comment:
10 blocks should be enough to yield 2 text elements from most live blogs.
Articles will generally have 1 body block.

Reply:
non-text element = e.g. third party embed? feels like 10 blocks might be excessive.

alternatively we can use another field like trailtext as the description.

Reply:
Yep - this is fairly generous over fetching.

We're happy to take single digit too many if it means we don't under fetch and don't get called out for not having enough text.
Fetching a maximum of 10 considered better than regularly fetching 70 or 80. With the chance to tweak this down once the concept has been accepted and approved.

Reply:
Likewise for not moving to just the standfirst or trailtext.
Using the body blocks to avoid having to justify a noticable change to the user visible output. We want the size reduction but don't want to getting into a larger discussion able formats just yet.

Reply:
Okay, I buy this - approving